integrations/postgresql/src/main/kotlin/com/xebia/functional/xef/vectorstores/postgresql/postgres.kt

```diff
@@ -11,14 +11,14 @@ enum class PGDistanceStrategy(val strategy: String) {
 }
 
 val createCollections: String =
-  """CREATE TABLE xef_collections (
+  """CREATE TABLE IF NOT EXISTS xef_collections (
```

Comment:
What's the reason of having this here instead of a migration file?

Reply:
Those vals were a thing I discovered when doing this PR. I'm with you, they need to be in a migration file. I don't know if addressing that in a further PR or if it's better to change it now.

Reply:
I would go for a different PR